### DISCUSSION TITLE
Companion crashes when closing the simulator

### DISCUSSION THREAD

**Discussion by SmilingCow** [EdgeTX/edgetx#5705] [Q&A]
Is it normal ?
I reinstalled Companion, same result.
Thank you for any info.

**Comment by elecpower** [comment]
It is not normal.
Please log an issue and provide as much information as possible to assist in triaging.

**Reply by SmilingCow** [reply]
I finally had to:
Desintall/reinstall Cimpanion = PB remains 
desintall appli and settings + reinstall= OK now
👍👍
I purged also registration keys related to companion 2.6 and 2.8.

All is OK now.
Thank you for your help.
